owl/SOMA.owl

```diff
@@ -148,7 +154,25 @@ AnnotationPropertyRange(SOMA:symbol xsd:string)
 #   Object Properties
 ############################
 
-# Object Property: <http://www.ease-crc.org/ont/SOMA.owl#after> (<http://www.ease-crc.org/ont/SOMA.owl#after>)
+# Object Property: <http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> (is initial scene of)
+
+AnnotationAssertion(rdfs:comment <http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> "A relation between StateTransitions and Scenes, which identifies the scene the transition starts from.")
+AnnotationAssertion(rdfs:label <http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> "is initial scene of")
+SubObjectPropertyOf(<http://www.ease-crc.org/ont/SOMA-SAY.owl#isInitialSceneOf> <http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#isObjectIncludedIn>)
```

Comment:
isInitialSceneOf has as domain and range Situations (a Scene and a StateTransition to be precise). Way back when, several DUL versions ago, Situations were a kind of SocialObject, so it made sense to have isObjectIncludedIn as a superproperty. However, Situations are now their own  thing, not necessarily objects, and so it would be nicer to directly subsume hasInitialScene to isSettingFor and likewise for the inverse property.

Reply:
Since I am ending up as requesting changes anyway, I would suggest that the isInitial/FinalScene and their inverses be moved from the includesObject and its inverse, and as originally suggested, have isSettingFor and its inverse be the superproperties.